.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+# This triggers whenever a tagged release is pushed
+name: Compile Assets and Create Draft Release
+
+on:
+  push:
+    tags:
+      - 'v*' # Trigger on tags beginning with 'v'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Install Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.x'
+          channel: 'stable'
+
+      - name: Install Melos
+        run: flutter pub global activate melos
+
+      - name: Install dependencies and compile assets
+        run: melos prepare
+
+      # Create the Draft release using the tag name
+      - name: Create Draft Release
+        id: create_release
+        uses: actions/create-release@v1
```

Comment:
It looks like this was archived in 2021 perhaps we should rather use one of the ones mentioned in the archived repo?

Reply:
Good catch! I updated this to a newer action that does both release creation and asset uploading.